extensions/blocks/rating-star/icon.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+export const StarBlockIcon = () => {
+	return (
+		<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+			<Path d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4V6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z" />
+		</SVG>
+	);
+};
+
+const getColor = props => ( props && props.color ? props.color : 'currentColor' );
+const getClassName = props => ( props && props.className ? props.className : '' );
+
+export const StarIcon = props => {
```

Comment:
Is there a risk that `props` doesn't exist? You could default it to empty object:

```js
( props: {} ) => { ... }
```

You could also [set a default value when destructuring from an object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values_2), like so:

```js
const { className: '' } = props;
```

Reply:
I think this is fine as it is.